.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# By default, the mathlib maintainers own everything in the repo.
+# Later matches will take precedence over this match.
+*	@leanprover-community/mathlib-maintainers
+
+src/tactic/		@leanprover-community/mathlib-meta
+
+src/category_theory/	@leanprover-community/mathlib-CT
+src/algebraic_topology/ @leanprover-community/mathlib-CT # for now the category theory team can take care of this
+
+src/algebraic_geometry/ @leanprover-community/mathlib-AG
```

Comment:
```suggestion
src/algebraic_geometry/ @leanprover-community/mathlib-AG
src/combinatorics/ @leanprover-community/mathlib-COMBI
```

Reply:
I think Johan wants to use arXiv abbreviations, so `mathlib-CO` would be better.

Reply:
Sorry I had no idea what it was!